### DISCUSSION TITLE
Allow projects to be transferred to another nhost account

### DISCUSSION THREAD

**Discussion by geodox** [nhost/nhost#72] [Ideas]
I'm a developer developing an application for a company. I'd really like the ability to transfer the project to an account they create when it's finished.

If this isn't possible, would it be possible to export the project that they could then import it instead of having to create the database etc all over again in their account?

**Comment by elitan** [comment]
This will be addressed in Nhost v2 where we use workspaces and it will be much easier to transfer projects between workspaces.
